mongodb-panache-quickstart/src/main/java/org/acme/mongodb/panache/Person.java

```diff
@@ -1,31 +1,28 @@
 package org.acme.mongodb.panache;
 
-import java.time.LocalDate;
-import java.util.List;
-
-import org.bson.codecs.pojo.annotations.BsonProperty;
-
 import io.quarkus.mongodb.panache.MongoEntity;
 import io.quarkus.mongodb.panache.PanacheMongoEntity;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+
+import java.time.LocalDate;
+import java.util.List;
```

Comment:
I' not sure about these import changes.
Are you using Quarkus code formatter ?
Did you run `mvn process-sources` ?

Reply:
I think that it's not available on ```quarkus-quickstarts```. But i resolved this issue